app/src/main/java/com/google/android/ground/ui/compose/ConfirmationDialog.kt

```diff
@@ -20,27 +20,58 @@ import androidx.compose.material3.AlertDialog
 import androidx.compose.material3.Text
 import androidx.compose.material3.TextButton
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.google.android.ground.ExcludeFromJacocoGeneratedReport
 import com.google.android.ground.R
+import com.google.android.ground.ui.theme.AppTheme
 
 @Composable
 fun ConfirmationDialog(
   @StringRes title: Int,
   @StringRes description: Int,
-  @StringRes dismissButtonText: Int = R.string.cancel,
-  @StringRes confirmButtonText: Int,
-  signOutCallback: () -> Unit,
-  dismissCallback: () -> Unit,
+  @StringRes dismissButton: Int = R.string.cancel,
+  @StringRes confirmButton: Int,
```

Comment:
technically we are passing text only, so why `dismissButtonText` -> `dismissButton`?

Reply:
Fixed